tests/app/celery/test_scheduled_tasks.py

```diff
@@ -488,3 +494,40 @@ def test_remove_csv_files_filters_by_type(mocker, sample_service):
     assert s3.remove_job_from_s3.call_args_list == [
         call(job_to_delete.service_id, job_to_delete.id),
     ]
+
+
+def test_delete_dvla_response_files_older_than_seven_days_removes_old_files(notify_api, mocker):
+    single_page_s3_objects = [{
+        "Contents": [
+            single_s3_object_stub('bar/foo1.txt', datetime_in_past(days=8)),
+            single_s3_object_stub('bar/foo2.txt', datetime_in_past(days=8)),
+        ]
+    }]
+    mocker.patch(
+        'app.celery.scheduled_tasks.s3.get_s3_bucket_objects', return_value=single_page_s3_objects[0]["Contents"]
+    )
+    remove_s3_mock = mocker.patch('app.celery.scheduled_tasks.s3.remove_s3_object')
+
+    delete_dvla_response_files_older_than_seven_days()
+
+    remove_s3_mock.assert_has_calls([
+        call(current_app.config['DVLA_RESPONSE_BUCKET_NAME'], single_page_s3_objects[0]["Contents"][0]["Key"]),
+        call(current_app.config['DVLA_RESPONSE_BUCKET_NAME'], single_page_s3_objects[0]["Contents"][1]["Key"])
+    ])
+
+
+def test_delete_dvla_response_files_older_than_seven_days_does_not_remove_files(notify_api, mocker):
+    single_page_s3_objects = [{
+        "Contents": [
+            single_s3_object_stub('bar/foo1.txt', datetime_in_past(days=6)),
+            single_s3_object_stub('bar/foo2.txt', datetime_in_past(days=9)),
```

Comment:
Could refactor the test to explicitly show what the values passed in for the days argument mean, as 9 doesn't mean anything until you look at the code and find out that there is a 2 day tolerance from the `older_than` date.